specs/derivation.md

```diff
@@ -875,11 +875,17 @@ safe head.
 
 The following fields of the derived L2 payload attributes are checked for equality with the L2 block:
 
-- `parent_hash`
-- `timestamp`
-- `randao`
-- `fee_recipient`
-- `transactions_list` (first length, then equality of each of the encoded transactions, including deposits)
+- Bedrock, Canyon, Delta, Ecotone Blocks
+  - `parent_hash`
+  - `timestamp`
+  - `randao`
+  - `fee_recipient`
+  - `transactions_list` (first length, then equality of each of the encoded transactions, including deposits)
+  - `gas_limit`
+- Canyon, Delta, Ecotone Blocks
+  - `withdrawals` (first presence, then length, then equality of each of the encoded withdrawals)
+- Ecotone Blocks
+  - `parent_beacon_block_root`
```

Comment:
Has the parent beacon block root been present since canyon? I thought it was new since ecotone

Reply:
It's new in Ecotone, the sections are the blocks in which it's validated in. So for parent beacon block root, just Ecotone. While Withdrawals was added for Canyon/Delta/Ecotone (we never added it to the specs previously).